tools/flash_algo/extract.py

```diff
@@ -44,6 +44,9 @@ def main():
                         help="Name of target to generate algo for")
     parser.add_argument("--all", action="store_true",
                         help="Build all flash algos for devcies")
+    parser.add_argument("--local-pack", dest="local_pack",
+                        default=None,
+                        help="Build all flash algos for devcies")
```

Comment:
It was either wrongly merged or firstly commited as it is now. We shall corrected it. I added you, should be able to push